tests/functional/a/arguments.py

```diff
@@ -261,3 +261,20 @@ def func(one, two, three):
 
 
 CALL = lambda *args: func(*args)
+
+
+# Ensure `too-many-function-args` is not emitted when a function call is assigned
+# to a class attribute inside the class where the function is defined.
+# Reference: https://github.com/PyCQA/pylint/issues/6592
+class FruitPicker:
```

Comment:
Would this raise before ? I suppose we need more (4 if we look above) arguments to trigger the message anyway ?

Reply:
Sorry if I misunderstand your question but let's say if I checkout `main` & copy the class as-is & run Pylint on it in an example.py, we get the error for each of these two lines in the test:
```python
pick_apple = _pick_fruit("apple")
pick_pear = _pick_fruit("pear")
```

Before the fix, these calls were seen as having two arguments each (one extra for an implicit `self`), and the code was comparing these arguments to the single `fruit` parameter from the definition of `_pick_fruit`.

Reply:
Ho, yes, my bad. I've mistaken this check with ``too-many-arguments``